app/src/main/java/io/appium/uiautomator2/handler/FirstVisibleView.java

```diff
@@ -78,7 +78,7 @@ protected AppiumResponse safeHandle(IHttpRequest request) throws UiObjectNotFoun
         }
 
         String id = UUID.randomUUID().toString();
-        AndroidElement androidElement = getAndroidElement(id, firstObject, null);
+        AndroidElement androidElement = getAndroidElement(id, firstObject, false);
```

Comment:
Should not here be `true`?

Reply:
ah... visible element might be multiple. we cannot say it is an exactly single element, so false...?

Reply:
exactly